ci-operator/README.md

```diff
@@ -5,6 +5,26 @@ This document describes how to create CI jobs for Openshift components using
 ci-operator and is intended for component developers who want to add tests to
 their CI process.
 
+Conventions
+-----------
+
+Under this directory, we have three main directories:
+
+*  `config/$repo/$branch.json`
```

Comment:
`config/$org/$repo/$branch.json`


specify this _must_ be the actual thing being tested -- if you're forking a k8s repo, use `openshift/kubernetes-$repo`